sds/src/scanner/mod.rs

```diff
@@ -1036,17 +1039,21 @@ mod test {
 
         let mut content = SimpleEvent::Map(BTreeMap::from([
             (
-                "message".to_string(),
+                "a-match".to_string(),
+                SimpleEvent::String("bcdef".to_string()),
```

Comment:
I would keep the exact same input for `a-match` and `z-match` so the only difference is the path
```suggestion
                SimpleEvent::String("abcdef".to_string()),
```

Reply:
Fixed in a9d44ba1848f996b99c3d8a4bb83e59d8af51ba6